content/docs/components/metadata.md

```diff
@@ -150,31 +149,12 @@ artifact in the **Artifact Store** on the Kubeflow UI.
 
 
 
-## Backend and REST API
-
-The Kubeflow metadata backend uses [ML Metadata
-(MLMD)](https://github.com/google/ml-metadata/blob/master/g3doc/get_started.md) 
-to manage the metadata and relationships. 
-
-The backend exposes a 
-[REST API](/docs/reference/metadata/v1alpha1/kubeflow-metadata-api-spec/).
+## GRPC backend
 
-You can add your own metadata types so that you can log metadata for custom
-artifacts. To add a custom type, send a REST API request to the
-[`artifact_types` endpoint](/docs/reference/metadata/v1alpha1/kubeflow-metadata-api-spec/#operation--api-v1alpha1-artifact_types-post). 
+The Kubeflow metadata deploys the [gRPC service](https://github.com/google/ml-metadata/blob/master/ml_metadata/proto/metadata_store_service.proto) of [ML Metadata
+(MLMD)](https://github.com/google/ml-metadata/blob/master/g3doc/get_started.md) to manage the metadata and relationships.
 
-For example, The following request registers an artifact type with 
-_name_ `myorg/mytype/v1` and three _properties_: 
-
-* `f1` (string)
-* `f2` (integer)
-* `f3` (double)
-
-```
-curl -X POST http://localhost:8080/api/v1alpha1/artifact_types \
-  --header "Content-Type: application/json" -d \
-  '{"name":"myorg/mytype/v1","properties":{"f1":"STRING", "f2":"INT", "f3": "DOUBLE"}}'
-```
+Kubeflow Metadata SDK saves and retrieves data via the gRPC service. Similarly, you can define your own metadata types to log and view metadata for your custom artifacts. For Python examples, you can check [MLMD Python client](https://pypi.org/project/ml-metadata/) and Kubeflow Metadata SDK [source code](https://github.com/kubeflow/metadata/blob/master/sdk/python/kubeflow/metadata/metadata.py). For Go examples, you can check the [source code](https://github.com/kubeflow/metadata/blob/master/watcher/handlers/metalogger.go) of the [resource watcher](#metadata-watcher).
```

Comment:
Do you have bandwidth to change "resource watcher" to "metadata watcher" since that is what it is referred to in the prior section?

Reply:
Done.